setting.yaml

```
@@ -0,0 +1,94 @@
+apiVersion: v1alpha1
+kind: Setting
+metadata:
+  name: halo-setting-casper
+spec:
+  forms:
+    - group: general
+      label: 基础设置
+      formSchema:
+        - $formkit: text
+          name: cover
+          label: 首页背景图
+          value: ""
+        - $formkit: text
+          name: links_cover
+          label: 友情链接页面背景图
+        - $formkit: text
+          name: archives_cover
+          label: 归档页面背景图
```

```suggestion
        - $formkit: attachment
          name: cover
          label: 首页背景图
          value: ""
        - $formkit: attachment
          name: links_cover
          label: 友情链接页面背景图
        - $formkit: attachment
          name: archives_cover
          label: 归档页面背景图
```
图片选择建议使用 attachment 组件。